client/src/redux/info/infoReducer.js

```diff
@@ -1,10 +1,12 @@
+import shuffle from 'lodash/shuffle';
+
 import * as actionTypes from '../actionTypes';
 import * as constants from '../../constants';
 
 export const defaultState = {
   tu: constants.INITIAL_TU,
   gameStatus: constants.GAME_STATUS_LOBBY,
-  startingRows: [],
+  availableRows: shuffle(Array(constants.GRID_SIZE).fill(null).map((_, i) => (i))),
```

Comment:
this can be slightly simplified to `availableRows: shuffle([...Array(constants.GRID_SIZE).keys()]),`

This makes an empty Array of size `constants.GRID_SIZE`, then spreads the keys (indexes) of that array into a new array.